Gracefully handle missing user menus, refs #13113

djjuhasz opened the pull request:
- Fall back to old i18n labels if login, logout, or myProfile menus have
been deleted or renamed.
- Simplify display logic in template

djjuhasz committed:
Gracefully handle missing user menus, refs #13113

- Fall back to old i18n labels if login, logout, or myProfile menus have
been deleted or renamed.
- Simplify display logic in template

jraddaoui reviewed (approved):
Nice clean-up along the way, thanks!

djjuhasz committed:
Use elseif for exclusive condition, refs #13113

Don't evaluate isAuthenticated() if the initial condition has already
determined that the user is not authenticated.

djjuhasz commented:
@jraddaoui I made one minor tweak to the conditional logic, can you please review again?

jraddaoui reviewed (approved):
:+1:

djjuhasz commented:
Merged 5bc0435ecca3156081d835849781b02fa5a7da90
